src/components/Player.vue

```diff
@@ -306,6 +308,11 @@ export default {
         this.setSeekTime(time);
       }
     },
+    onAudioError(event) {
+      if (event.srcElement.networkState === 3) {
```

Comment:
The server will return a 404 if there is no audio file attached. I would only skip in that case, since if it is another error, the error is much more catastrophic (and probably every track will be skipped). Also let the user know that an error happened (and disambiguate between the case where there is just no audio file and we skipped to the next track and the case where a much more catastrophic error happened).

Reply:
The problem is that the `onerror` event doesn't include the http response code. The only to way to know the server returned a 404 is by launching the event again, which doesn't seem like a clean solution to me

Reply:
The more catastrophic error would either be (1) the client is not connected to the internet (2) the server is down. Maybe we should alert the user if either of those is the case and prevent playing (since it wouldn't make sense)?